src/agent/onefuzz-agent/src/tasks/config.rs

```diff
@@ -33,6 +33,8 @@ pub struct CommonConfig {
     pub heartbeat_queue: Option<Url>,
 
     pub telemetry_key: Option<Uuid>,
+
+    pub setup_dir: Option<PathBuf>,
```

Comment:
Is it ever valid for `setup_dir` to be absent? Doesn't every task require access to `target_exe`, which will always occur within a `setup_dir`?

Reply:
I was thinking about keeping the backward compatibility with the way we currently run the tasks by not requiring an additional parameter. 
I still have to make the default value being the parent directory of target_exe

Reply:
Yes.  In the existing debugging mode as well as the upcoming `local fuzzing` mode, it means having to specify an additional argument that is often not useful locally.

Reply:
changed this be non optional